moveit_py/moveit/servo_client/README.md

```diff
@@ -0,0 +1,9 @@
+# Servo Module
+This module provides a client for interacting with MoveIt servo.
+
+# Tutorial
+Add links to official tutorials here.
```

Comment:
Again, might be tricky to keep these in sync, and may not be necessary to link them from the source code?

Reply:
Agreed updating now.

Reply:
https://github.com/ros-planning/moveit2/pull/1546/commits/9f24297bd0a5f237b3c93c245950545c53a32ce3